sekayasin/play-ah-backend-zeus
# #16216316 Users should receive a JWT on successful Registration

**What does this PR do?**
Generates a JSON Web Token after a user has provided the cortrect credentials during registration or login.
**Description of Task to be completed?**
- write tests
- add user token generation functionality

**How should this be manually tested?**
- Open POSTMAN, add the link `/api/users/` to the localhost url to register a new user.
- Provide the body format for registration from the documentation
`{
  "user":{
    "username": "Jacob",
    "email": "jake@jake.jake",
    "password": "jakejake"
  }
}`
User should be able to receive a JSON Web Token after both registration and login.

**What are the relevant pivotal tracker stories?**
#16216316

**Screenshots (if appropriate)**
<img width="972" alt="screen shot 2018-12-03 at 3 17 32 pm" src="https://user-images.githubusercontent.com/9946845/49373320-bc90a900-f70e-11e8-95bd-6bcea0af7176.png">
